backend/src/routes/api/components/list.ts

```diff
@@ -22,13 +22,19 @@ export const listComponents = async (
 
   // Fetch the enabled config maps
   const enabledCMs = await getEnabledConfigMaps(fastify, applicationDefs);
-  const getCSVForApp = (app: ODHApp) =>
+  const getCSVForApp = (app: ConsoleApplication) =>
     operatorCSVs.find(
-      (operator) => app.spec.csvName && operator.metadata.name.startsWith(app.spec.csvName),
+      (operator) => app.spec.csvName && operator.metadata?.name?.startsWith(app.spec.csvName),
     );
 
   // Get the components associated with the installed KfDefs or operators
   const installedComponents = applicationDefs.reduce((acc, app) => {
+    app.spec.isEnabled = true;
```

Comment:
I believe we will still need to check for the enabled status of some of the applications that use the validation CMs to determine the status.

Reply:
yes, i think you're right. I'll keep any isEnabled checks if I've removed them. I think I also inadvertentlyu removed code related to [features](https://github.com/red-hat-data-services/odh-dashboard/blob/master/data/features.json) that i'll fix.